credential/exchange/model.go

```diff
@@ -288,6 +288,7 @@ type Filter struct {
 	Type             string        `json:"type,omitempty"`
 	Format           string        `json:"format,omitempty"`
 	Properties       interface{}   `json:"properties,omitempty"`
+	Required         []string      `json:"required,omitempty"`
```

Comment:
can we add `additionalProperties` which is a boolean value too?

Reply:
ok yup I'll throw it in there